docs/_news/news07.md

```diff
@@ -20,13 +20,15 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Nodejs, Liberty and Spring projects. 
+- Effortlessly [visualise data](https://github.com/eclipse/codewind/issues/977) like your CPU useage, memory footprint and web traffic on our included application monitoring dashboard at the click of a button. If you don't want or need it, you can simply [disable it](https://github.com/eclipse/codewind/issues/1290). We'll host the dashboard, so displaying the graphs won't eat into your project's resources.
```

Comment:
I think "visualise" with the "s" is the British version. Please change to "visualize" with the "z" for the American spelling. :)

Side note: We use American spellings in the docs, but if a British team member is writing blogs, I leave the British spellings and punctuation intact.

Reply:
- The Oxford comma strikes again--please add a comma after "footprint."

Reply:
I try to avoid first person pronouns like "I" and "we" and "our." So, I'd just say, "the included application..." and "Codewind hosts the dashboard..."

For things like blogs, first person is OK because blogs generally have an author, and readers know who the "I" is referring to. :)